Libraries/LibCrypto/Curves/Ed448.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2024, Altomani Gianluca <altomanigianluca@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/ScopeGuard.h>
+#include <LibCrypto/Curves/Ed448.h>
+#include <LibCrypto/OpenSSL.h>
+
+#include <openssl/core_names.h>
+#include <openssl/evp.h>
+
+namespace Crypto::Curves {
+
+ErrorOr<ByteBuffer> Ed448::generate_private_key()
```

Comment:
Non-maintainer take: I think we should avoid fallible returns for things that _most likely_ won't fail.  Only because if they do fail, it's either us using the API incorrectly or something completely out of our control like a memory allocation failure. 
Failing fast in those scenarios seems better than bubbling it up to callers. 

Slight bonus is that call sites of these functions become a little less noisy. Anyway, this could just return `ByteBuffer` and `MUST(...)` any fallible calls it makes IMO.

Reply:
Honestly, I am of the opposite idea: since error handling comes at a little cost we should try to cover every case to preserve uniformity and future proofing.